lark/lexer.py

```diff
@@ -9,16 +9,25 @@
 )
 from types import ModuleType
 import warnings
+try:
```

Comment:
Already defined below

Reply:
No, this is the import, below is only the check for the standalone parser so that that doesn't crash with a NameError. I guess I should add a comment below.

Reply:
But why not only keep the one below?

Reply:
I mean, move this one to below

Reply:
Because I don't think that we want to import `interegular` for the standalone parser.

Reply:
Okay, fair enough. It would add unnecessary load time.